docs/extend/extend-apiml/authentication-for-apiml-services.md

```diff
@@ -129,40 +136,48 @@ Authentication via the login endpoint is performed in the following way:
 
 ![Zowe client certificate authentication diagram](../../images/api-mediation/zowe-client-cert-auth.png)
 
-### Procedure to authenticate with client certificates
-
-Follow these steps to authenticate with client certificates:
+#### Prerequisites
```

Comment:
Maybe there should be an introductory line:

"Make sure the following requirements are met before setting up client certificate authentication"